pkg/k8s/synced/crd.go

```diff
@@ -105,7 +105,8 @@ func AllCiliumCRDResourceNames() []string {
 	return append(
 		AgentCRDResourceNames(),
 		CRDResourceName(v2.CEWName),
-		CRDResourceName(v2alpha1.CNCName),
+		CRDResourceName(v2.CNCName),
```

Comment:
Do you need to wait for both versions? I don't think so. Just v2 should work here.

Reply:
As said before,  v2alpha1 configurations are not retrieved in that case!